From 19e598d8126a3ccfe1b33a58aac940358928800c Mon Sep 17 00:00:00 2001
From: Pirmin Vogel <vogelpi@lowrisc.org>
Date: Tue, 11 Feb 2020 17:26:21 +0100
Subject: [PATCH 2/5] [prim] An important change

We do NOT want to port this back to OT.

Signed-off-by: Pirmin Vogel <vogelpi@lowrisc.org>

diff --git a/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv b/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
index 2bb3d70..978775a 100644
--- a/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
+++ b/hw/ip/prim/rtl/prim_ram_2p_async_adv.sv
@@ -57,6 +57,8 @@ module prim_ram_2p_async_adv #(
   input [CfgW-1:0] cfg_i
 );
 
+  // Another important change
+
   // Calculate ECC width
   localparam int ParWidth  = (EnableParity) ? 1 :
                              (!EnableECC)   ? 0 :
-- 
2.17.1

